Promise.resolve(3).then(res => console.log(res))

Promise.reject("Boo!").catch(res => console.log(res))